src/ModalHeader.js

```diff
@@ -4,24 +4,35 @@ import classnames from 'classnames';
 
 class ModalHeader extends React.Component {
 
+  renderCloseButton() {
+    if (this.props.closeButton === true) {
```

Comment:
that will give a false positive for the new case where it's an element won't it?